project.css

```diff
@@ -275,10 +370,44 @@ a {
   padding: 12px 80px;
 }
 
+.return-button:hover {
+  background-color: var(--purple-hover);
+}
+
 .continue-button {
   padding: 12px 65px;
 }
 
+.continue-button:hover {
+  background-color: var(--black-hover);
+}
+
+.category-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
**Question**: Is this needed even after centering things through flex?

Reply:
Apparently; the longer one ("sports and outdoors") doesn't center without it.

Reply:
Ah! That makes sense, thanks.